guide/getting-started/getting-started.md

```diff
@@ -34,70 +34,68 @@ Illustration sources
 
 # Getting started
 
-Welcome to the Bitcoin Design Guide! You’re in for an adventure. This chapter is all about the big concepts that are helpful to be familiar with when creating Bitcoin products. You’ll find a summary of concepts on this page, click through if you would like to dig deeper into a specific topic. The guide isn't a comprehensive overview of all Bitcoin topics, but only the ones you should know as a designer working in this space.
-
 ---
 
 ### [Why Bitcoin is unique]({{ '/guide/getting-started/why-bitcoin-is-unique/' | relative_url }})
 
-The internet has opened an infinite design space for information and communication. Bitcoin adds the new dimension of transferring value that could bring similar change as previous waves of innovation.
+Familiarize yourself with some of the most significant concepts that bitcoin designers need to know.
 
 ---
 
 ### [Why design for Bitcoin]({{ '/guide/getting-started/why-design-for-bitcoin/' | relative_url }})
 
-Bitcoin is fundamentally open and participatory and allows designers and creators to explore and shape experiences for users around the globe.
+Bitcoin is open-source, decentralized, permissionless, fungible, stateless, and non-custodial. If those words don’t mean anything to you, consider starting here.  
```

Comment:
The page is not about explaining those terms (we have the principles page for that), but why those properties make designing for Bitcoin exciting - why the reader should drop everything they are doing and start designing Bitcoin stuff. So to me, the new summary doesn't capture the page intent/content well.

Reply:
Reverted in https://github.com/BitcoinDesign/Guide/pull/221/commits/7aaf56ee1b2506f5293704ee23824570ab259633